packages/fxa-auth-server/test/remote/account_status_tests.js

```diff
@@ -48,12 +48,34 @@ describe('remote account status', function() {
         )
         .then(
           (response) => {
+            assert.deepEqual(response.authenticationMethods, ['pwd', 'email']);
+            assert.equal(response.authenticatorAssuranceLevel, '1');
             assert.equal(response.locale, 'en-US', 'locale is stored');
           }
         );
     }
   );
 
+  it(
+    'account status includes locale when authenticated',
```

Comment:
nit: this test and its partner below are now asserting more things than just "locale", so the test description is inaccurate.